app/test/std_sdk_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/celestiaorg/celestia-app/app"
 	"github.com/celestiaorg/celestia-app/app/encoding"
 	"github.com/celestiaorg/celestia-app/pkg/user"
+	util "github.com/celestiaorg/celestia-app/test/util"
```

Comment:
I don't expect an import alias to be needed here b/c the alias name is the same as the package name

```suggestion
	"github.com/celestiaorg/celestia-app/test/util"
```